### DISCUSSION TITLE
Cannot read properties of null (reading 'startsWith') [May 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2279] [Discord Threads]
> 🙋‍♂️ **mahmouds12**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Looks like the error is happening when the shareid is invalid  
#  
> 💁 **o1lab**  
#### you could try reentering a valid one and in the meanwhile we will do the improvements  
#  
> 💁 **o1lab**  
#### to handle these errors  
#  
> 🙋‍♂️ **mahmouds12**  
#### what is a shareid? and where to enter it?  
#  
> 👨‍🦲 **dstala**  
#### you can find details here   
#### https://docs.nocodb.com/setup-and-usages/import-airtable-to-sql-database-within-a-minute-for-free/#retrieve-share-base-id--url  
#  
> 🙋‍♂️ **mahmouds12**  
#### aah thanks I was  writing the "base id" not the URL  
#  
> 🙋‍♂️ **mahmouds12**  
#### Now I am getting this error while syncing    
#### type "double" does not exist   
####     at Parser.parseErrorMessage (...\nocodb\packages\nocodb\node_modules\pg-protocol\src\parser.ts:369:69)  
#  
> 💁 **o1lab**  
#### have DMed  
#  
> 💁 **o1lab**  
#### Please try the latest release v0.91.1   
####    
#### We've made improvements  
#  
> 🙋‍♂️ **mahmouds12**  
#### great thanks, the schema is migrated but got a new error about views   
####  Syncing views   
####  :: [1/19] Grid : RFQ / Pending RFQs WEB   
####  viewColumnUpdate - viewer : Not allowed  
#  
> 💁 **o1lab**  
#### Ahh  
#  
> 💁 **o1lab**  
#### Let me check with the team  
#  
> 👨‍🦲 **dstala**  
#### <@847339595314364467> DM'd for more details.  
#  
> 💁 **o1lab**  
#### We re managing to migrate all schema from the base you had shared with us.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/978700470187094036)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
